src/parameters/parameters_linear_solver.cpp

```diff
@@ -7,6 +7,18 @@ void LinearSolverParam::declare_parameters (dealii::ParameterHandler &prm)
 {
     prm.enter_subsection("linear solver");
     {
+        prm.declare_entry("atol", "0.001",
```

Comment:
Could these be moved to the ode_solver parameters section rather than this one? I think they would be more relevant there.